controls/srg_ctr/SRG-APP-000092-CTR-000165.yml

```diff
@@ -3,4 +3,165 @@ controls:
   levels:
   - medium
   title: {{{ full_name }}} must initiate session auditing upon startup.
-  status: pending
+  rules:
+  - cluster_logging_operator_exists
```

Comment:
typo: the rule is called `cluster_logging_operator_exist` (no `s`). Please also add the SRG reference to the rule.